app/models/research_plan.rb

```diff
@@ -98,15 +99,27 @@ def svg_files
     svg_files
   end
 
+  def clone_existing_attachments(attachments, attachment_id, current_user_id)
```

Comment:
It seems that the attachments are copied in terms of a database entry but not on the filesystem which leads to the problem i mentioned in the comments section